src/localization/successor_manager/node.h

```diff
@@ -57,16 +57,15 @@ bool operator==(const Node &lhs, const Node &rhs);
 // custom specialization of std::hash can be injected in namespace std
 // This makes a node hashable to use for std::unordered_set<Node>
 namespace std {
```

Comment:
As of lately the best practices ask not to put anything into the `std` namespace to avoid any form of collision. It would be better to declare a has function here and use it as an explicit parameter to whichever container type that uses it.